### PR TITLE
LPS-115096 Sort annotations alphabetically

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/components/_field-options.scss [hunk 1]
```diff
@@ -41,9 +41,13 @@ $c: '.ddm-field-options';
 				background: transparent;
 				border: none;
 				color: #6b6c7e;
-				cursor: default;
+				cursor: auto;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/components/_field-options.scss:44]: @alinedoleron, `auto` is the default value of the `cursor` property, right? Therefore, we can remove this line. 😉 

In addition, why didn't you squash this commit with the first one? Let's commit less putting together very related commits, ok?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js [hunk 2]
```diff
@@ -177,7 +177,9 @@ class Sidebar extends Component {
 					defaultLanguageId,
 					editingLanguageId,
 					keywordReadOnly:
-						field.fieldName == 'options'
+						field.fieldName == 'options' ||
+						field.fieldName == 'rows' ||
+						field.fieldName == 'columns'
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js:182]: @alinedoleron, please sort it alphabetically. 😉

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 3]
```diff
@@ -1084,8 +1084,8 @@ definition {
 			rowEntry = "Created Form Name");
 	}
 
-	@ignore = "true"
 	@description = "This is a use case for LPS-98073."
+	@ignore = "true"
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:1088]: @alinedoleron, why didn't you squash this commit with the third one? Let's commit less putting together very related commits, ok?

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
new pr: https://github.com/natocesarrego/liferay-portal/pull/655

**Pull request closed without merging**